
From: Roger Luethi <rl@hellgate.ch>

I really wanted /proc/pid/statm to die and I still believe the
reasoning is valid.  As it doesn't look like that is going to happen,
though, I offer this fix for the respective documentation.  Note: lrs/drs
fields are switched.

Signed-off-by: Roger Luethi <rl@hellgate.ch>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/Documentation/filesystems/proc.txt |   20 +++++++++++---------
 1 files changed, 11 insertions(+), 9 deletions(-)

diff -puN Documentation/filesystems/proc.txt~fix-proc-pid-statm-documentation Documentation/filesystems/proc.txt
--- 25/Documentation/filesystems/proc.txt~fix-proc-pid-statm-documentation	Thu Aug  5 15:31:53 2004
+++ 25-akpm/Documentation/filesystems/proc.txt	Thu Aug  5 15:31:53 2004
@@ -169,16 +169,18 @@ information. The  statm  file  contains 
 process memory usage. Its seven fields are explained in Table 1-2.
 
 
-Table 1-2: Contents of the statm files 
+Table 1-2: Contents of the statm files (as of 2.6.8-rc3)
 ..............................................................................
- File     Content                         
- size     total program size              
- resident size of memory portions         
- shared   number of pages that are shared 
- trs      number of pages that are 'code' 
- drs      number of pages of data/stack   
- lrs      number of pages of library      
- dt       number of dirty pages           
+ Field    Content
+ size     total program size (pages)		(same as VmSize in status)
+ resident size of memory portions (pages)	(same as VmRSS in status)
+ shared   number of pages that are shared	(i.e. backed by a file)
+ trs      number of pages that are 'code'	(not including libs; broken,
+							includes data segment)
+ lrs      number of pages of library		(always 0 on 2.6)
+ drs      number of pages of data/stack		(including libs; broken,
+							includes library text)
+ dt       number of dirty pages			(always 0 on 2.6)
 ..............................................................................
 
 1.2 Kernel data
_
